include/cantera/oneD/refine.h

```diff
@@ -45,14 +46,25 @@ class Refiner
         m_npmax = npmax;
     }
 
+    size_t getMaxPoints() const {
+        return m_npmax;
+    }
+
     //! Set the minimum allowable spacing between adjacent grid points [m].
     void setGridMin(double gridmin) {
         m_gridmin = gridmin;
     }
 
+    //! @deprecated To be removed after Cantera 2.3
```

Comment:
I'm not sure that changing `gridMin` to `getGridMin` is an improvement in consistency. The pattern of having `foo()` and `setFoo(...)` as the getter/setter pair is used quite widely in Cantera.